app/controllers/api_new/payouts_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
I think we should just remove the magic comment whenever we come across them.

Reply:
`frozen_string_literal: true` improves performance slightly in some cases

I wish Ruby had moved to the default of frozen string literal by now and they're in the process of getting there but until then it probably should be in files.

Reply:
I believe it is no longer required in Ruby 3. And since the majority of the project already doesn't use it, no sense it adding it to /some/ files in the PR. My LOGAF is a 2 (of 5...)